### ISSUE TITLE
[Bug] small one. v4.3.0 release number  is not showing up in admin UI. still showing previous one v4.2.0

### ISSUE THREAD

**Issue by avillach** [awslabs/service-workbench-on-aws#893]
**Versions (please complete the following information):**
 - Release Version installed v4.3.0

![Screen Shot 2022-01-27 at 9 11 48 PM](https://user-images.githubusercontent.com/4821297/151475145-68a11cc6-94c8-43a4-b983-208a42d66808.png)

**Comment by gazzyt** [comment]
Just ran into this issue too. Also I note that the release tag for 4.3.0. is `crea` rather than `v4.3.0`

**Comment by Bingjiling** [comment]
Hi @avillach and @gazzyt , 

Thanks for reaching out! We'll look into fixing the version issues. 

Thanks,
Yanyu

**Comment by nguyen102** [comment]
Thanks for reaching out. We've fixed the release tag to be `v4.3.0`.

As for the release version number. Can you check the bottom of your staging file for the field `versionDate`? Please change that to `Latest Release Version: 4.3.0` and redeploy. That should update the version number of your SWB installation. 

```
# Version number of current release
versionNumber: 'Beta'

# Release date of current release
versionDate: 'Latest Release Version: 4.3.0'
```

**Comment by gazzyt** [comment]
Thanks @nguyen102 - unfortunately I don't have the bandwidth at the moment to test this with a re-deploy but I had a look at the changes made between the old tag and new tag and I would expect the `versionNumber` and `versionDate` in my `main/config/settings/$STAGE.yml` file to be automatically set to those values by the `get-release-info.sh` script (based on the changes I see in `CHANGELOG.md`.

Is v4.3.0 considered a beta release? Or would you expect versions like this?
```
# Version number of current release
versionNumber: '4.3.0'

# Release date of current release
versionDate: '2022-01-26'
```

**Comment by nguyen102** [comment]
We've released [v4.3.1](https://github.com/awslabs/service-workbench-on-aws/releases/tag/v4.3.1) to fix this issue. You can deploy the latest release to fix the versioning issue in the UI. 
![Screen Shot 2022-02-02 at 10 47 25 AM](https://user-images.githubusercontent.com/3661906/152187765-cf24a0be-354a-4131-ac30-fcefe87e2367.png)

**Comment by gazzyt** [comment]
Thanks @nguyen102 - I just installed v4.3.1 and the versioning all looks good.
